packages/react/src/components/Notification/Notification.tsx

```diff
@@ -418,6 +581,80 @@ ToastNotification.defaultProps = {
   timeout: 0,
 };
 
+/**
+ * InlineNotification
+ * ==================
+ */
+
+export interface InlineNotificationProps {
```

Comment:
should this extend Div attributes? extra props can be passed in through `..rest`